### ISSUE TITLE
The windows with different sizes corners drawn incorrectly.

### ISSUE THREAD

**Issue by llevkin** [Ezelia/EZGUI#4]
As I understand it, the calculation of the height of the corners of the window comes on top of a couple of corners. If the height of the upper corners of the window and bottom different - there is a bug ...

**Comment by alaa-eddine** [comment]
Can you please provide a screenshot or a live example.
not sure to understand what you mean :)

**Comment by llevkin** [comment]
![screen](https://cloud.githubusercontent.com/assets/1658843/7614859/4c11f4b2-f9a2-11e4-818f-85b48b6faf90.jpg)

Here... top and bottom corner and line have different height..

**Comment by alaa-eddine** [comment]
I see what you mean now :)
actually, there is no simple way to handle different corners sizes.
because if I calculate the background height taking in consideration corner-tr and corner-br height, there will be the same problem if left corner and right corner have different size ....

the problem becomes worst if all corners have different sizes . this is why I assume that they all have same width/height.

there is a possible solution I see here, it'll be by adding possibility to change background paddings in all directions and accept negative paddings. in your case, a negative bottom bgPadding can solve the problem.

don't know if this workaround can fix your issue temporarly, but you can try to set bgPadding to negative value, it's not very clean but can avoid you waiting for the fix.

**Comment by llevkin** [comment]
The task in which the four corners are of different sizes - can not be solved and absurd. The task in which the left and right angles are equal and the upper and lower heights are not equal - solved. Such a structure is quite common interface. I think you still need to choose the starting point for the bottom (left / right) and the top (left / right) to determine the angles of elevation.

**Comment by llevkin** [comment]
But this is my imho, of course)

**Comment by alaa-eddine** [comment]
yes this can be a solution, I'll see it don't add much complexity to the code.
it's important to stabilise the code in next updates so I can implement the new components : tabs, and text input are planed :)
